diffsmhm/diff_stats/cuda/sigma.py

```diff
@@ -269,8 +274,10 @@ def sigma_serial_cuda(
         Array of radial bin edges, note that this array is one longer than the
         number of bins in the 'rp' (xy radial) direction.
     zmax : float
-        Maximum distance to integrate over in the z direction. Note this should
-        be a whole number due to the unit binning of Corrfunc.
+        Maximum separation in the z direction. Particles with z distance less
+        than zmax from a given halo are included in surface density calculations
+        for that halo. Note this should be a whole number due to the unit
```

Comment:
Sorry. What I meant here is that zmax doesn't need to be a whole number for the cuda versions, right?

Reply:
ah yes! you're right